src/app/tests/suites/certification/Test_TC_DRLK_2_1.yaml

```diff
@@ -1551,20 +1551,292 @@ tests:
       response:
           value: NumberOfCredentialsSupportedPerUserValue
 
-    - label: "Cleanup the created user"
-      PICS: DRLK.S.F00 && DRLK.S.F07
+    - label: "Step 36: TH sends ClearCredential Command to DUT"
+      PICS: DRLK.S.F00 && DRLK.S.F07 && DRLK.S.C26.Rsp
+      command: "ClearCredential"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "Credential"
+                value: { CredentialType: 1, CredentialIndex: 1 }
+
+    - label:
+          "Step 37: TH sends ClearUser Command to DUT with the UserIndex as 1"
+      PICS: DRLK.S.F00 && DRLK.S.F07 && DRLK.S.C1d.Rsp
       command: "ClearUser"
       timedInteractionTimeoutMs: 1000
       arguments:
           values:
               - name: "UserIndex"
                 value: 1
 
-    - label: "Clean the created credential"
-      PICS: DRLK.S.F00 && DRLK.S.F07 && DRLK.S.C26.Rsp
-      command: "ClearCredential"
-      timedInteractionTimeoutMs: 1000
+    - label: "Step 38a: TH reads AliroReaderVerificationKey attribute from DUT"
+      PICS: PICS_SKIP_SAMPLE_APP && DRLK.S.F0d && DRLK.S.A0080
```

Comment:
Why is this PICS_SKIP_SAMPLE_APP?

For CI, the PICS for `DRLK.S.F0d` and `DRLK.S.A0080` are already false, so this step would be skipped anyway, no?

Similar elsewhere here: there should be no PICS_SKIP_SAMPLE_APP anywhere.

Reply:
yes removed the PICS_SKIP_SAMPLE_APP